sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -24,13 +24,12 @@
 test_path = os.path.dirname(os.path.abspath(__file__))
 modules_path = os.path.dirname(test_path)
 scripts_path = os.path.join(modules_path, "xcvrd")
-helper_file_path = os.path.join(scripts_path, "xcvrd_utilities"+"/y_cable_helper.py")
 sys.path.insert(0, modules_path)
 
 os.environ["XCVRD_UNIT_TESTING"] = "1"
-load_source('y_cable_helper', scripts_path + '/xcvrd_utilities/y_cable_helper.py')
```

Comment:
With the removal of this line, I believe we should also be able to remove `from imp import load_source` on line 7. I'm surprised LGTM didn't complain about an unused import.